### PR TITLE
SettingsPopover fixes

### PR DESCRIPTION
**Pull request by personnamedmike**
Screenshot 2023-08-16 at 10 45 14 AM

### CODE REVIEW

File: src/features/settings/utils/useFormattedDate.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export const useFormattedDate = (date: string | undefined) => {
```
Comment (gpxl-dev) [inline src/features/settings/utils/useFormattedDate.ts:1]: super minor but I prefer `date?: string`

File: src/features/settings/SettingsPopover.tsx [hunk 2]
```diff
@@ -1,41 +1,59 @@
...
+  };
 
-  useClickOutside(
-    settingsPopoverRef,
-    () => togglePopover(false),
-    ['click', 'keydown']
-  )
+  useClickOutside(settingsPopoverRef, () => togglePopover(false), [
+    "click",
+    "keydown",
```
Comment (gpxl-dev) [inline src/features/settings/SettingsPopover.tsx:42]: remove `keydown` - remember to test closing the modal with spacebar when focused on the button

**Review by gpxl-dev — APPROVED** [review]

File: src/features/settings/SettingsPopover.tsx (hunk 2 above)
Comment (personnamedmike) [reply src/features/settings/SettingsPopover.tsx]: removed

File: src/features/settings/utils/useFormattedDate.ts (hunk 1 above)
Comment (personnamedmike) [reply src/features/settings/utils/useFormattedDate.ts]: Fixed. This is less verbose and nice looking

**Pull request merged**